torchdynamo/codegen.py

```diff
@@ -110,6 +118,18 @@ def __call__(self, value, allow_cache=True):
                 self._create_load_const(graph_outputs[graph_outputs_key].index)
             )
             output.append(create_instruction("BINARY_SUBSCR"))
+            if isinstance(value, UnspecializedPrimitiveVariable):
```

Comment:
Are we baking in the assumption here that all UnspecializedPrimitiveVariable's are numpy?  Perhaps we should have a specific class for numpy.